scattering/van_hove.py

```diff
@@ -214,3 +214,113 @@ def compute_partial_van_hove(trj, chunk_length=10, selection1=None, selection2=N
         g_r_t += g_r_t_frame
 
     return r, g_r_t
+
+def atom_conc_from_list(atom_list, trj):
+	""" 
+    	Generates a dictionary of atom concentration with the key corresponding to the atoms in atom_list
+
+    	Parameters
+    	----------
+    	trj : mdtraj.Trajectory
+        	trajectory on which partial vhf were calculated form
+	atom_list : list, optional, default = None
+        	list of unique atoms. if left at default, atom_list generated from trj
+
+
+    	Return
+    	-------
+    	mf : dict
+        	dictionary containing concentration of atoms from atom_list
+    	"""
+	
+	topology = trj.topology
+	atoms = [i.element.symbol for i in topology.atoms]
+    	
+	#atom_list generation
+	if atom_list == None:
```

Comment:
The way the function is set up now, it might not make sense to have `atom_list` as an argument.  Instead it might make more sense to parse the element symbols from the keys of `partial_dict` and grab the atom form factor and concentration that way anyways.